runtime/src/lib.rs

```diff
@@ -358,6 +361,19 @@ pub fn handle_mailbox_commands(drivers: &mut Drivers) -> ! {
     // Indicator to SOC that RT firmware is ready
     drivers.soc_ifc.assert_ready_for_runtime();
     caliptra_drivers::report_boot_status(RtBootStatus::RtReadyForCommands.into());
+    // Disable attestation if in the middle of executing an mbox cmd during warm reset
+    if drivers.mbox.cmd_busy() {
+        let reset_reason = drivers.soc_ifc.reset_reason();
+        if reset_reason == ResetReason::WarmReset {
+            let mut result = DisableAttestationCmd::execute(drivers);
+            if let Ok(mut resp) = result {
```

Comment:
I think if this fails we want to fatal error

Reply:
Done.